src/store/ducks/auth/sagas.ts

```diff
@@ -38,10 +56,10 @@ function* authorize(user, password) {
 }
 
 export function* auth() {
-  while (true) {
-    const actions = yield take(AuthTypes.AUTH_REQUEST);
-    const { user, password } = actions.payload;
-    const token = yield fork(authorize, user, password)
-    console.log("TOKEN NO SAGA", token)
-  }
+    while (true) {
```

Comment:
E  esse rolê aqui, mano? :thinking:

Reply:
o saga usa generators, que é uma função que tem uma execução que não é contínua, esse while(true) garante que ela vai voltar pro início quando chegar no final

Reply:
Se vc ta falando huauhsuha demoro